crates/core/src/diff/morph.rs

```diff
@@ -469,7 +469,7 @@ impl Iterator for Morph<'_> {
                             Op::Patch(Patch::PrependBefore { before: from.node }),
                             // Set newly inserted node as append parent
                             Op::Patch(Patch::PushCurrent),
-                            Op::Append {
+                            Op::AppendSiblings {
```

Comment:
Such a small fix for such a long test.

Reply:
I can shrink it down to a minimal example. This is just forked over from the repro branch.

Reply:
Instead I will opt to remove this test when a cleaner end-to-end test is available from the playback fixtures i'm writing.